playbook/app/pb_kits/playbook/pb_multiple_users_stacked/multiple_users_stacked.html.erb

```diff
@@ -8,14 +17,39 @@
         text: "+#{object.users.count - object.display_count}",
         variant: "primary",
         rounded: true,
-        classname: "pb_multiple_users_stacked_item second_item" }) %>
+        classname: "pb_multiple_users_stacked_item second_item"
+      }) %>
     <% elsif object.bubble %>
-      <% object.users.slice(1,3).each_with_index do |item, idx| %>
-        <%= pb_rails("avatar", props: item.merge({size: "xs", classname: "pb_multiple_users_stacked_item #{idx == 0 ? "second_item#{object.triple_bubble ? " triple_bubble" : ""}#{object.quadruple_bubble ? " quadruple_bubble" : ""}" : idx == 1 ? "third_item#{object.quadruple_bubble ? " quadruple_bubble" : ""}" : "fourth_item"}", dark: object.dark}) ) %>
+      <% object.users.slice(1, 3).each_with_index do |item, idx| %>
```

Comment:
adding double_bubble and split this into multiple lines because line length was long